community-tutorials/web/01-en.md

```diff
@@ -0,0 +1,427 @@
+rm -r wordpress
+```
+Now create a database see step 1.7 <br>
+Now you can open Wordpress over https://DOMAIN/ and follow the shown steps
+
+## Step 2.2 - Installing YOURLS
+YOURLS is an open source URL shorter with a web GUI <br>
+Follow step 1.5 and create a new domain or subdomain for the website <br>
+Please replace "FOLDER" with the folder you set in step 1.5
+```sh
+cd /var/www/
+git clone https://github.com/YOURLS/YOURLS.git
+mv YOURLS FOLDER
+cd FOLDER
+```
+Now create a database see Step 1.7 <br>
+Now we need to configure YOURLS
+```sh
+cd user
+cp config-sample.php config.php
+nano config.php
+```
+Now you need to set the following settings (you need to replace it in the second ''): <br>
+Replace `'your db user name'` - with your database username name
+```
+/** MySQL database username */
+define( 'YOURLS_DB_USER', 'your db user name' );
+```
+Replace `'your db password'` - with your database password
+```
+/** MySQL database password */
+define( 'YOURLS_DB_PASS', 'your db password' );
+```
+Replace `'yourls'` - with your database name (username)
+```
+/** The name of the database for YOURLS
+ ** Use lower case letters [a-z], digits [0-9] and underscores [_] only */
+define( 'YOURLS_DB_NAME', 'yourls' );
+```
+Replace `'http://your-own-domain-here.com'` - with your URL
+```
+/** YOURLS installation URL
+ ** All lowercase, no trailing slash at the end.
+ ** If you define it to "http://sho.rt", don't use "http://www.sho.rt" in your browser (and vice-versa)
+ ** To use an IDN domain (eg http://héhé.com), write its ascii form here (eg http://xn--hh-bjab.com) */
+define( 'YOURLS_SITE', 'http://your-own-domain-here.com' );
+```
+Replace `'modify this text with something random'` - fill there in random letters and numbers
+```
+/** A random secret hash used to encrypt cookies. You don't have to remember it, make it long and complicated
+ ** Hint: copy from http://yourls.org/cookie */
+define( 'YOURLS_COOKIEKEY', 'modify this text with something random' );
+```
+Now you need to create a user and set its (secure) password - Replace `'username' => 'password', `
+```
+/** Username(s) and password(s) allowed to access the site. Passwords either in plain text or as encrypted hashes
+ ** YOURLS will auto encrypt plain text passwords in this file
+ ** Read http://yourls.org/userpassword for more information */
+$yourls_user_passwords = [
+	'username' => 'password',
+	// 'username2' => 'password2',
+	// You can have one or more 'login'=>'password' lines
+];
+```
+Now save your edit with <kbd>ctrl + x</kbd>, then press <kbd>y</kbd> and now save it with enter <br/>
+Now you can open YOURLS over https://DOMAIN/admin and follow the shown steps and login
+
+### Step 2.2.1 - Installing a mordern YOURLS Theme (Sleeky) - Optional
+Download the Source code (zip) from https://github.com/Flynntes/Sleeky/releases/latest and unpack it and upload the `sleeky-backend` folder of the folder from the archiv in the user/plugin folder of your YOURLS installation <br>
```

Comment:
Please add those commands to make it understandable.

Reply:
done